packages/core/src/sdk/deliveryPromise/useDeliveryPromise.ts

```diff
@@ -360,12 +360,19 @@ export function useDeliveryPromise({
               facet.values[pickupInPointFacetIndex] = pickupInPointFacet
             }
 
-            facet.values = facet.values.sort((a, b) =>
-              (a.value ?? '').localeCompare(b.value ?? '')
-            )
-
```

Comment:
I believe this sort is also important to stay, because otherwise sometimes can happen that a facet value appears before other and sometimes after, which causes a weird experience. Therefore, we should have a sort for the facet values and a sort for the DP facets blocks (which you added successfully)